content/docs/1.4/scalers/cron.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Cron"
+layout = "scaler"
+availability = "v1.0+"
+maintainer = "Community"
+description = "Scale applications based on Cron Schedules."
+go_file = "cron_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `cron` trigger that scales based on a Cron Schedule.
+
+```yaml
+triggers:
+- type: cron
+  metadata:
+    # Required
+    timezone: <time zone the cron should be run>
+    start: <start of cron>
+    end: <end of cron>
+    metricName: replicacount
```

Comment:
Can you elaborate on what this is for? Simply for the HPA or?

Reply:
Yes simply for the HPA. Would you recommend to not define it but just use a default name for the purpose of HPA?

Reply:
What do you recommend @zroubalik Do we do this convention based internally?

Frankly, as a user, I don't want to worry about it and just define the schedule

Reply:
Yes, I would expose this to the user (all the things about metrics names will be part of https://github.com/kedacore/keda/issues/733), but for now it is not needed to expose to the users.

Reply:
removed the metric